build_tools/release_automation.py

```diff
@@ -62,7 +62,7 @@
         {'name': 'Potrzebowski, Wojciech','affiliation': 'Data Management and Software Centre, European Spallation Source ERIC', 'orcid': '0000-0002-7789-6779'},
         {'name': "Prescott, Stewart",'affiliation': 'University of New South Wales'},
         {'name': 'Ferraz Leal, Ricardo','affiliation': 'Oak Ridge National Laboratory'},
-        {'name': 'Rozycko, Piotr','affiliation': 'Data Management and Software Centre, European Spallation Source ERIC' },
+        {'name': 'Rozycko, Piotr','affiliation': 'Data Management and Software Centre, European Spallation Source ERIC', 'orcid' : '0000-0002-2359-1013' },
```

Comment:
Since you added orcid.. can you also add `z` in my last name? :) (sorry)

Reply:
Ops, what a shame! I am sorry I didn't spot it before. We probably need to go through entire list again.